apps/authentication/models/connection_token.py

```diff
@@ -317,4 +334,17 @@ def expire_at(self):
         return (timezone.now() + timezone.timedelta(days=365)).timestamp()
 
     def is_valid(self):
-        return True
+        return super().is_valid()
+
+    @classmethod
+    def get_user_permed_account(cls, user, asset, account_name, protocol):
+        """
+        管理员 token 可以访问所有资产的账号
+        """
+        with tmp_to_org(asset.org_id):
+            account = asset.accounts.filter(name=account_name).first()
+            if not account:
+                return None
+        account.actions = ActionChoices.all()
+        account.date_expired = timezone.now() + timezone.timedelta(days=5)
+        return account
```

Comment:
This code looks fine so far. No significant changes should be made to it.

However, I would like to make one minor improvement: replace `super()` with `self` when accessing attributes that belong to the current object in Django, which could improve readability.

The updated code is:

```python
class Account(models.Model):

    # ... other fields ...

    @staticmethod
    def create_from_web(request):
        ...
```

Do you want to proceed with this change?
  
Thank you very much for letting me know!